packages/presentation/src/components/ObjectPopup.svelte

```diff
@@ -189,7 +189,7 @@
     }
   }
 
-  $: updateLocation(scrollDiv, selectedDiv, objects, selected)
+  $: updateLocation(scrollDiv, selectedDiv)
```

Comment:
You not fixed the issue.

updateLocation should be called with onscroll, not sure how it worked before.

![image](https://user-images.githubusercontent.com/477235/210036623-386a3a75-61d1-41c2-a767-8e737543ba02.png)

Reply:
Also updateLocation still should be updated if objects are changed, this could be achived by passing objects to updateLocation, whats why it was passed before.

Reply:
Fixed, by this way:   
<div class="scroll" on:scroll={() => updateLocation(scrollDiv, selectedDiv, objects, selected)} bind:this={scrollDiv}>